### DISCUSSION TITLE
关于使用文档

### DISCUSSION THREAD

**Discussion by LiShouHai** [Tencent/tdesign-vue-next-starter#409] [Q&A] [answered]
想问一下 现在有可使用的中文文档嘛

**Comment by timi137137** [comment, marked as answer]
https://tdesign.tencent.com/starter/docs/vue-next/get-started

**Reply by LiShouHai** [reply]
为啥我这403呢

**Reply by LiShouHai** [reply]
最近在看你封装的 axios  悠静萝莉 （手动狗头

**Reply by timi137137** [reply]
> 为啥我这403呢

刷新缓存看看

**Reply by LiShouHai** [reply]
不知道其他人怎么样 我这还是403 （。

**Reply by timi137137** [reply]
> 不知道其他人怎么样 我这还是403 （。

你给个截图？

**Comment by LiShouHai** [comment]
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/71856739/217419203-e31d74e7-bcf7-4e12-81c4-c1f23ab41c30.png">
不好意思 一直没带图

**Reply by timi137137** [reply]
@uyarn 来看一眼，估计是cos权限对于一些地区没开

**Reply by uyarn** [reply]
其他浏览器也这样吗

**Reply by LiShouHai** [reply]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/71856739/217433733-61b7731f-1a3c-4881-90d7-a83e9e80b70b.png">
换了一个 Safari 解决了 谢谢大佬 读文档去了
